backend/clickhouse/logs.go

```diff
@@ -36,6 +37,13 @@ var logKeysToColumns = map[modelInputs.ReservedLogKey]string{
 	modelInputs.ReservedLogKeyEnvironment:     "Environment",
 }
 
+// These keys show up as recommendations, but with no recommended values due to high cardinality
+var defaultLogKeys = []*modelInputs.QueryKey{
```

Comment:
should `body` be part of this since we are dropping the mv?

Reply:
Currently the body key is never being used since it is the default key and being filtered out on the frontend. However, I'll probably add this back in another PR for customizing columns on the logs table